eras/conway/impl/cardano-ledger-conway.cabal

```
@@ -71,7 +71,7 @@ library
         -Wunused-packages
 
     build-depends:
-        base >=4.14 && <4.19,
+        base >=4.14 && <4.30,
```

Where did `30` come from?
```suggestion
        base >=4.14 && <5,
```